src/timeseriesflattener/feature_spec_objects.py

```diff
@@ -203,38 +274,101 @@ def __eq__(self, other):
-
-    entity_id_col_name: str = "entity_id"
-    # Col name for ids in the input dataframe.
-
-    loader_kwargs: Optional[dict] = None
-
-    # Optional keyword arguments for the data loader
+    """The minimum specification required for all
+        collapsed time series. (temporal features), whether looking ahead or behind.
+        Mostly used for inheritance below.
+
+    Fields:
+        values_loader (Optional[Callable]):
+            Loader for the df. Tries to resolve from the data_loaders registry,
+            then calls the function which should return a dataframe.
+        values_name (Optional[str]):
+            A string that corresponds to a key in a dictionary of multiple
+            dataframes that corresponds to a name of a type of values.
+        loader_kwargs (Optional[dict]):
+            Optional kwargs passed onto the data loader.
+        values_df (Optional[DataFrame]):
+            Dataframe with the values.
+        feature_name (str):
+            The name of the feature. Used for column name generation, e.g.
+            <prefix>_<feature_name>.
+        prefix (str):
+            The prefix used for column name generation, e.g.
+            <prefix>_<feature_name>.
+        input_col_name_override (Optional[str]):
+            An override for the input column name. If None, will  attempt
+            to infer it by looking for the only column that doesn't match id_col_name
+            or timestamp_col_name.
+        output_col_name_override (Optional[str]):
+            Override the generated column name after flattening the time series
+        interval_days (Union[int, float]):
+            How far to look in the given direction (ahead for outcomes,
+            behind for predictors)
+        resolve_multiple_fn (Callable):
+            A function used for resolving multiple values within the
+            interval_days.
+        key_for_resolve_multiple (Optional[str]):
+            Key used to lookup the resolve_multiple_fn in the
+            resolve_multiple_fns registry. Used for column name generation. Only
+            required if you don't specify a resolve_multiple_fn. Call
+            timeseriesflattener.resolve_multiple_fns.resolve_multiple_fns.get_all()
+            for a list of options.
+        fallback (Union[Callable, int, float, str]):
+            Which value to use if no values are found within interval_days.
+        allowed_nan_value_prop (float):
+            If NaN is higher than this in the input dataframe during
+            resolution, raise an error. Defaults to: 0.0.
+        entity_id_col_name (str):
+            Col name for ids in the input dataframe. Defaults to: entity_id."""
+
+    class Doc:
+        short_description = """The minimum specification required for all 
+        collapsed time series. (temporal features), whether looking ahead or behind.
+        Mostly used for inheritance below."""
```

Comment:
```suggestion
    class Doc:
        short_description = """The minimum specification required for all 
        collapsed time series. (temporal features), whether looking ahead or behind.
        Mostly used for inheritance below."""
```

When is the `Doc` class used?

Reply:
We also talked about this yesterday, but for future reference. The `Doc` class is used to generate the top part (the description) of the docstring. Will add a note on how to modify the docstrings correctly